spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,173 @@
+                                :officemanager=>"ELIZABETH CASSIDY",
+                                :daysopen=>"Monday - Friday - 8:00 to 5:00, Last Saturday of the Month  - 8:00 to 12:00",
+                                :daysclosed=>"Monday - Friday from 1:00 to 1:30",
+                                :holidaysclosed=>
+                                "Christmas Day (12/25/23), New Year's Day (1/1/24), Martin Luther King Jr. Day (1/15/24), Lincoln's Birthday (2/12/24), President's Day (2/19/24), Truman's Birthday (5/8/24), Memorial Day (5/27/24), Juneteenth (6/19/24), Independence Day (7/4/24), Labor Day (9/2/24), Columbus Day (10/14/24), Veterans Day (11/11/24), Thanksgiving Day (11/28/24), Christmas Day (12/25/24)",
+                                :additionaldaysclosed=>"3/6/2023 (at 3:45 PM ),    3/6/2023 (at 9:30 AM until 12:00 PM ),    3/6/2023 (open at 12:00 PM ),  7/1/2023,   7/3/2023,     11/24/23,        11/25/23, 11/29/2024, 12/24/2024, ",
+                                :latlng=>{:latitude=>"37.837063", :longitude=>"-94.347818"},
+                                :facebook_url=>"https://m.facebook.com/nevadalicenseoffice",
+                                :additional_license_office_info=>"MOTOR VEHICLE PHONE IN RENEWAL - (417) 448-5705",
+                                :":@computed_region_ny2h_ckbz"=>"313",
+                                :":@computed_region_c8ar_jsdj"=>"73",
+                                :":@computed_region_ikxf_gfzr"=>"2218"},
+                              {:number=>"059",
+                                :dorregionnumber=>"5",
+                                :type=>"1MV",
+                                :name=>"FERGUSON-OFFICE CLOSED UNTIL FURTHER NOTICE",
+                                :address1=>"10425 WEST FLORISSANT",
+                                :city=>"FERGUSON",
+                                :state=>"MO",
+                                :zipcode=>"63136",
+                                :located_in=>"Personalized plates can be picked up at the North County License Office. ",
+                                :county=>"St. Louis County",
+                                :phone=>"(314) 733-5316",
+                                :fax=>"(314) 733-5319",
+                                :size=>"3",
+                                :email=>"FERGUSONAGENTOFFICE@DOR.MO.GOV",
+                                :latlng=>{:latitude=>"38.737935", :longitude=>"-90.220029"},
+                                :textingphonenumber=>"(314) 730-0606",
+                                :additional_license_office_info=>"Personalized plates can be picked up at the North County License office located at 11700 W Florissant, St. Louis, MO 63033.",
+                                :":@computed_region_ny2h_ckbz"=>"203",
+                                :":@computed_region_c8ar_jsdj"=>"51",
+                                :":@computed_region_ikxf_gfzr"=>"2210"}]
+  end
```

Comment:
A common practice is to put this stubbed data into another file in your specs though, so it's not creating clutter in your specs. Usually a fixtures directory will hold this stubbed info, then you can read in the file from your spec.